src/Dapr.Actors.AspNetCore/RouterBuilderExtensions.cs

```diff
@@ -23,6 +23,15 @@ public static void AddDaprConfigRoute(this IRouteBuilder routeBuilder)
             });
         }
 
+        public static void AddDaprHealthzRoute(this IRouteBuilder routeBuilder)
+        {
+            routeBuilder.MapGet("healthz", async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status200OK;
+                await context.Response.CompleteAsync();
+            });
+        }
```

Comment:
Not sure if it's useful to you or not, but we have a health checks package (more bells and whistles). https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-3.1

This makes the health checks extensible via DI - with lots of community extensions here: https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/tree/master/src

Reply:
@rynowak Thanks, will look at it!